provisioning/device/test/_polling_state_machine_test.js

```diff
@@ -129,15 +154,6 @@ describe('polling state machine', function () {
         });
       });
 
-
-      /* Tests_SRS_NODE_PROVISIONING_TRANSPORT_STATE_MACHINE_18_014: [ If `PollingTransport.registrationRequest` succeeds with status==Assigned, it shall emit an 'operationStatus' event and  call `callback` with null, the response body, and the protocol-specific result. ] */
-      it('and fires an operationStatus event if it succeeds', function (testCallback) {
```

Comment:
>it [](start = 6, length = 2)

this test corresponds to the emit() call you removed in the polling state machine.

Reply:
Kind of the point.

---
In reply to: [282193838](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282193838) [](ancestors = 282193838)